pythonapp/app.py

```diff
@@ -0,0 +1,22 @@
+import os
```

Comment:
Why adding it in the root folder? Any reason we need this particular app to validate the CPU, memory and goroutine utilization of the sidecar?

Reply:
These kinds of things probably should be in a sub folder of the `.github` folder.

Reply:
Yes but I don't think we even need a custom app. Maybe even fortio or some pre-baked image can be used as the app.

Reply:
From what it looks like so far ,the app is needed to get metrics using the port it enables dapr metrics logging at